docs/content/stable/yugabyte-platform/security/enable-encryption-in-transit.md

```diff
@@ -4,7 +4,7 @@ headerTitle: Enable Encryption in Transit (TLS)
 linkTitle: Enable encryption in transit (TLS)
 description: Use Yugabyte Platform to enable encryption in transit (TLS) on a YugabyteDB universe and connect to clients.
 menu:
-  stable:
+  latest:
```

Comment:
Is this correct? Page not showing in menu under stable

Reply:
There is no separate "How to rotate" subsection for every cert type, but it's described in the procedure. (the developer who reviewed the doc when the content was modified for 2.8 agreed that it's fine)